test/functional/test_runner.py

```diff
@@ -322,6 +319,16 @@ def main():
         print("Rerun ./configure with --enable-wallet, --with-utils and --with-daemon and then make")
         sys.exit(0)
 
+    if not (len(ALL_SCRIPTS) == dict.fromkeys(ALL_SCRIPTS).keys()):
+        print("{}WARNING!{} There are duplicate tests in the test list:".format(BOLD[1], BOLD[0]))
+        duplicates = set()
+        for test in ALL_SCRIPTS:
+            if ALL_SCRIPTS.count(test) > 1:
+                duplicates.add(test)
+
+        for dup in duplicates:
+            print("- ", dup)
```

Comment:
`print` itself prints a space between arguments so that you can write `print("-", dup)`.